src/tactic/command/login_group_trigger.py

```diff
@@ -46,14 +47,36 @@ def execute(my):
 
         login_group = login_group_name.lower()
         login_group = login_group.replace(" ", "_")
+        login_group_code = sobj.get_value('login_group')
 
         sobj.set_value('login_group', login_group)
         sobj.set_value('code', login_group)
 
         sobj.commit(triggers=False)
 
+        my.update_related(login_group, login_group_code)
 
+    def update_related(my, login_group, prev_login_group):
 
+        login_in_groups = Search.eval("@SOBJECT(sthpw/login_in_group['login_group','%s'])" % prev_login_group)
 
+        if login_in_groups:
+
+            server = TacticServerStub.get()
+            login_in_group_dict = {}
+            
+            data = {
+                "login_group": login_group
+            }
+
+            for login_in_group in login_in_groups:
+                login_in_group_code = login_in_group.get("code")
+                login_in_group_sk = server.build_search_key("sthpw/login_in_group", login_in_group_code)
+                login_in_group_dict[login_in_group_sk] = data
+
+            try:
+                server.update_multiple(login_in_group_dict)
+            except Exception:
+                raise Exception
```

Comment:
It should have an error msg:

except Exception, e:
          raise TacticException('Error updating login_in_group %s' %e.__str__())